### PR TITLE
[11.0][FIX] account_credit_control: Missing read access

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Supersedes https://github.com/OCA/account-financial-tools/pull/727.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza what do you think?

**Comment by mreficent** [comment]
@pedrobaeza what do you think?

**Review by rven — APPROVED** [review]

File: account_credit_control/models/credit_control_run.py [hunk 1]
```diff
@@ -140,8 +140,7 @@ def _generate_credit_lines(self):
             else:
                 report += _(
                     "Policy \"<b>%s</b>\" has not generated any "
-                    "Credit Control Lines.<br/>" % policy.name
-                )
+                    "Credit Control Lines.<br/>") % policy.name
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_run.py:143]: I didn't notice this mistake in translations. Can you redo translations for not requiring to re-translate this?

Comment (MiquelRForgeFlow) [reply account_credit_control/models/credit_control_run.py:143]: Isn't it automatically fixed by @oca-travis ?

Comment (pedrobaeza) [reply account_credit_control/models/credit_control_run.py:143]: I don't think so as the msgid is changing. Can you confirm if so?

Comment (MiquelRForgeFlow) [reply account_credit_control/models/credit_control_run.py:143]: Well, the only way to proof or disproof my claim is just merging it and wait, and if it is not fixed automatically, then open new PR to fix that. My basis is in https://github.com/OCA/knowledge/commit/2c04cf8566f564b60881839015cd1ab2b6fc13dd, it's an automatically generated commit fixing the `.pot` file

Comment (MiquelRForgeFlow) [reply account_credit_control/models/credit_control_run.py:143]: Also you can clearly see in https://github.com/OCA/account-financial-tools/blob/11.0/account_credit_control/i18n/account_credit_control.pot#L1056 (and the other `.po` files) that there is nothing to fix :S

Comment (pedrobaeza) [reply account_credit_control/models/credit_control_run.py:143]: OK, that's the kind of check I wanted you to do. We can merge then. Thanks!

**Pull request merged**